tests/phpunit/src/Commands/Api/ApiCommandTest.php

```diff
@@ -21,7 +21,6 @@ class ApiCommandTest extends CommandTestBase {
 
   public function setUp(): void {
     parent::setUp();
-    $this->clientProphecy->addOption('headers', ['Accept' => 'application/json']);
```

Comment:
This is necessary to kill an escaped mutant. We need to check that this method is called, but it isn't actually called on every test case.